### PR TITLE
(GH-28) added `gradle.exe` to the list of possible `GradlePlainExecutable`

### LINKED ISSUE #28: Cake.Gradle does not work, if gradle was installed globally using chocolatey
currently, if `gradle` is installed on the system and no `gradlew` is present in the project either `gradle.bat` or `gradle` are run (depending on windows/no windows).

If `gradle` was installed using [chocolatey](https://chocolatey.org/) `gradle` is present, however it is available via a `gradle.exe`-shim and so `Cake.Gradle` reports `Gradle Runner: Could not locate executable.`

### PR DESCRIPTION
**Pull request by nils-a**
also added some tests around ToolPath and WorkingDirectory to prepare for modifications.

fixes #28

### CODE REVIEW

File: src/Cake.Gradle.Tests/GradleRunnerFixture.cs [hunk 1]
```diff
@@ -1,40 +1,44 @@
...
+using Cake.Testing.Fixtures;
 
 namespace Cake.Gradle.Tests
 {
     public class GradleRunnerFixture : ToolFixture<GradleRunnerSettings>
     {
-        private GradleLogLevel? _gradleLogLevel;
-        private string _task;
-        private string _arguments;
+        private GradleRunner tool;
```
Comment (gep13) [inline src/Cake.Gradle.Tests/GradleRunnerFixture.cs:8]: Did we want to follow the `_` convention that is already in place here?

Comment (gep13) [reply src/Cake.Gradle.Tests/GradleRunnerFixture.cs:8]: If not, we should probably change the others to not have the `_`.

**Review by gep13 — CHANGES_REQUESTED** [review]

File: src/Cake.Gradle.Tests/GradleRunnerFixture.cs (hunk 1 above)
Comment (nils-a) [reply src/Cake.Gradle.Tests/GradleRunnerFixture.cs:8]: @gep13 good point. I planned on enabling that, but somehow it failed and I did not even notice.

Comment (nils-a) [reply src/Cake.Gradle.Tests/GradleRunnerFixture.cs]: @gep13 I modified `.editconfig` accordingly and fixed this.

**Review by gep13 — APPROVED** [review]
LGTM

**Comment by gep13** [comment]
@nils-a your changes have been merged, thanks for your contribution 👍

**Pull request merged**